include/bios_handler.hpp

```diff
@@ -60,6 +62,24 @@ class IbmBiosHandler : public BiosHandlerInterface
      * @param[in] i_msg - The callback message.
      */
     virtual void biosAttributesCallback(sdbusplus::message_t& i_msg);
+
+  private:
+    /**
+     * @brief API to read given attribute from BIOS table.
+     *
+     * @param[in] attributeName - Attribute to be read.
+     * @return - Bios attribute current value.
+     */
+    types::BiosAttributeCurrentValue
+        readBiosAttribute(const std::string& attributeName);
+
+    /**
+     * @brief API to process "hb_field_core_override" attribute.
+     *
+     * The API checks value stored in VPD. If found default then the BIOS value
+     * is saved to VPD else VPD value is restored in BIOS attribute.
+     */
+    void processFieldCoreOverride();
```

Comment:
can we also log an event when we back it up to VPD and VPD did not have a default value with the old and new values on VPD side

Reply:
restore from VPD is to the pending table or current table ?

Reply:
> restore from VPD is to the pending table or current table ?

Its pending attributes, will update that.